liquibase-dist/src/main/archive/GETTING_STARTED.txt

```diff
@@ -117,29 +117,18 @@ Running the update command allowed your developer database to match the defined
 
 In this example, we will add a new changeSet to create a "works for" column in the persons table.
 
-To add this changeSet, open the sample changelog file in your existing editor, and uncomment the following changeSet:
+To add this changeSet, open the sample changelog file in your existing editor, and copy + paste one of the following changeSet examples:
 
   XML changelog example code
   ---------------------------
-    <changeSet id="3" author="your.name">
-        <addColumn tableName="person">
-            <column name="worksfor_company_id" type="int"/>
-        </addColumn>
-    </changeSet>
     <changeSet id="4" author="your.name">
-        <addForeignKeyConstraint constraintName="fk_person_worksfor"
-                                 baseTableName="person" baseColumnNames="worksfor_company_id"
-                                 referencedTableName="company" referencedColumnNames="id"/>
+        <addForeignKeyConstraint constraintName="fk_person_worksfor" baseTableName="person" baseColumnNames="worksfor_company_id" referencedTableName="company" referencedColumnNames="id"></addForeignKeyConstraint>
     </changeSet>
 
```

Comment:
We can't just take out these previous changeSets because the ones you left rely on them. 

If you want them to only add one changeSet, you'll have to get rid of 4 not 3

Reply:
Kept both, and renumbered to 4 and 5 per Mario.